src/encoded/types/modification.py

```diff
@@ -26,7 +26,7 @@ class Modification(Item):
         'constructs.tags',
         'constructs.designed_to_target',
         'modified_regions.aliases',
-        'target_of_mod.target_summary'
+        'target_of_mod.display_title'
```

Comment:
Is this necessary? My impression is that display_title of a linkTo gets embedded by default, along with @id and @type etc.